bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/LedgerDescriptorImpl.java

```diff
@@ -116,7 +117,7 @@ ByteBuf getExplicitLac() {
      * @param journal log the fence entry in the Journal
      * @return A future which will be satisfied when add entry to journal complete
      */
-    private SettableFuture<Boolean> logFenceEntryInJournal(Journal journal) {
+    private SettableFuture<Boolean> logFenceEntryInJournal(Journal journal) throws InterruptedException {
```

Comment:
This method returns a Future, I wonder if it is okay to throw the Interrupted Exception or to return a failed Future

Reply:
Good point. will fix